cluster/examples/kubernetes/ceph/csi/example/cephfs/storageclass.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: csi-cephfs
-provisioner: cephfs.csi.ceph.com
+provisioner: rook-ceph.cephfs.csi.ceph.com
```

Comment:
please update the document about how these provisioners names are generated, so if users deploy operator in different namespace should be able to update this as well.

Reply:
The examples are already updated. The docs don't currently mention these explicitly so I didn't change it.